tests/integration/targets/ec2_placement_group/tasks/env_cleanup.yml

```
@@ -1,5 +1,5 @@
 - name: remove any instances in the test VPC
```

```suggestion
- name: Remove any instances in the test VPC
```
Could you please update tasks names with a uppercase letter?